lib/config.js

```diff
@@ -4,6 +4,7 @@ module.exports = function() {
   var defaultConfig = {
     host: '127.0.0.1',
     port: 1337,
+    schemename: 'http',
```

Comment:
This should either be in camel case (i.e., `schemeName`), or just be named `scheme`.  Although I tend to prefer the latter, I have no big opinion on which one we go for.